### ISSUE TITLE
albanode.initialize_disk ReadTimeout: disk is initialized but cannot claimed.

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework#848]
While performing a setup on the GIG environment, we saw this error. 
It appears that `node.client.add_disk` times out. The disk is initialized & mounted but no ASD is available...

This is pretty annoying because we have to remove the disk and re-add it again to try again.

```
2016-08-24 16:09:18 77300 +0200 - stor-02.be-g8-3 - 32277/140384358926144 - celery/celery.worker.job - 385 - ERROR - Task albanode.initialize_disk[185ef319-6a0a-4b56-abdd-1edbf3af4be7] raised unexpected: ReadTimeout(ReadTimeoutError('None: None',),)
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 240, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 438, in __protected_call__
    return self.run(*args, **kwargs)
  File "/opt/OpenvStorage/ovs/lib/albanodecontroller.py", line 119, in initialize_disks
    result = node.client.add_disk(disk_id)
  File "/opt/OpenvStorage/ovs/extensions/plugins/asdmanager.py", line 101, in add_disk
    return self._call(requests.post, 'disks/{0}/add'.format(disk_id), timeout=60)
  File "/opt/OpenvStorage/ovs/extensions/plugins/asdmanager.py", line 56, in _call
    response = method(**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/api.py", line 107, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/api.py", line 53, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 468, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 576, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/adapters.py", line 449, in send
    raise ReadTimeout(e, request=request)
ReadTimeout: None: None
```

**Comment by khenderick** [comment]
Seems this one was fixed by openvstorage/framework-alba-plugin#219

**Comment by JeffreyDevloo** [comment]
### Steps
- Setup a cluster
- Added diskroles
- Added backend
- Initialized disk

### Output

Initialization took longer than the timeout time before https://github.com/openvstorage/framework-alba-plugin/pull/219 (which was 60s).

### Additional information

The timeout in https://github.com/openvstorage/framework-alba-plugin/pull/219 was a hot fix for this particular problem as well

### Test result

Test passed.

### Packages
- openvstorage                       2.7.4-rev.4159.0707c77-1                                 amd64        openvStorage
- openvstorage-backend               1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin
- openvstorage-backend-core          1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps       1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin Web Applications
- openvstorage-cinder-plugin         1.2.2-rev.38.dcc3b76-1                                   amd64        OpenvStorage Cinder plugin for OpenStack
- openvstorage-core                  2.7.4-rev.4159.0707c77-1                                 amd64        openvStorage core
- openvstorage-hc                    1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin HyperConverged
- openvstorage-sdm                   1.6.4-rev.388.1f4375d-1                                  amd64        Open vStorage Backend ASD Manager
- openvstorage-webapps               2.7.4-rev.4159.0707c77-1                                 amd64        openvStorage Web Applications
